app/src/pages/inside/uniqueErrorsPage/constants.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const RP_CLUSTER_LAST_RUN = 'rp.cluster.lastRun';
```

Comment:
Could you please comment on this constant. I don't see any description on it in the task.

Reply:
we have an state when analysis was ran, but no error logs have been found. we have different text '... But if launch scope has been changed, you can re-run analysis one more time'